src/transformers/benchmark/benchmark.py

```diff
@@ -204,10 +255,27 @@ def encoder_forward():
 
                 return memory, summary
             else:
+
+                if not self.args.no_tpu and is_tpu_available():
+                    # run additional 10 times to stabilize compilation for tpu
+                    logger.info("Do inference on TPU. Running model 10 times to stabilize compilation")
```

Comment:
@LysandreJik - For TPU I ran the function 10 times before measuring to stabilize compilation. Should I do the same for torchscript?